src/ai_marketplace_monitor/telegram.py

```
@@ -0,0 +1,152 @@
+        logger: Logger | None = None,
+    ) -> bool:
+        url = f"https://api.telegram.org/bot{self.telegram_bot_token}/sendMessage"
+
+        # Helper function to format text based on message_format
+        def format_bold(text: str) -> str:
+            if self.message_format == "html":
+                return f"<b>{text}</b>"
+            elif self.message_format == "markdown":
+                return f"*{text}*"
+            else:
+                return text
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)